README.md

```diff
@@ -76,8 +76,6 @@ ClusterAPI provider OpenStack images and manifests are published after every PR
 * With a Google Cloud account you can get a quick overview [here](https://console.cloud.google.com/storage/browser/artifacts.k8s-staging-capi-openstack.appspot.com/components)
 * The manifests are available under:
   * [master/infrastructure-components.yaml](https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/components/master/infrastructure-components.yaml): 
-    latest build from the main branch, overwritten after every merge
```

Comment:
I think we should keep this:
1. the doc is still valid as the nightly build still exists
2. I think we should keep the nightly build as it's still useful

Reply:
Fixed.